### PR TITLE
fix lite doc, test=document_fix

### PR DESCRIPTION
**Pull request by LDOUBLEV**

### CODE REVIEW

File: deploy/lite/readme.md [hunk 1]
```diff
@@ -123,18 +123,27 @@ cd build.opt/lite/api/
 下面以PaddleOCR的超轻量中文模型为例，介绍使用编译好的opt文件完成inference模型到Paddle-Lite优化模型的转换。
 
 ```
-# 下载PaddleOCR的超轻量文inference模型，并解压
+# 可以下载PaddleOCR V1.1版本的inference模型，V1.1比1.0效果更好，模型大小更小
```
Comment (tink2123) [inline deploy/lite/readme.md:126]: 推荐下载PaddleOCR V1.1版本的inference模型，效果更好，模型更小

File: deploy/lite/readme.md [hunk 2]
```diff
@@ -123,18 +123,27 @@ cd build.opt/lite/api/
...
+wget  https://paddleocr.bj.bcebos.com/20-09-22/mobile-slim/det/ch_ppocr_mobile_v1.1_det_prune_infer.tar && tar xf  ch_ppocr_mobile_v1.1_det_prune_infer.tar
+wget  https://paddleocr.bj.bcebos.com/20-09-22/mobile-slim/rec/ch_ppocr_mobile_v1.1_rec_quant_infer.tar && tar xf  ch_ppocr_mobile_v1.1_rec_quant_infer.tar
+# 转换V1.1检测模型
+./opt --model_file=./ch_ppocr_mobile_v1.1_det_prune_infer/model  --param_file=./ch_ppocr_mobile_v1.1_det_prune_infer/params  --optimize_out=./ch_ppocr_mobile_v1.1_det_prune_opt --valid_targets=arm
+# 转换V1.1识别模型
+./opt --model_file=./ch_ppocr_mobile_v1.1_rec_quant_infer/model  --param_file=./ch_ppocr_mobile_v1.1_rec_quant_infer/params  --optimize_out=./ch_ppocr_mobile_v1.1_rec_quant_opt --valid_targets=arm
+
+
+# 或下载使用PaddleOCR的V1.0超轻量文inference模型，并解压
```
Comment (tink2123) [inline deploy/lite/readme.md:135]:
```suggestion
# 或下载使用PaddleOCR的V1.0超轻量inference模型，并解压
```

File: deploy/lite/readme.md (hunk 1 above)
Comment (LDOUBLEV) [reply deploy/lite/readme.md]: Done, Thx for big old

File: deploy/lite/readme.md (hunk 2 above)
Comment (LDOUBLEV) [reply deploy/lite/readme.md]: Done, thx for big old

**Pull request merged**